docs/admin.rst

```diff
@@ -79,157 +104,264 @@ Upgrade Documentation`_ on how to upgrade the drives.
+   the passphrase and/or two-factor authentication secret in your own password
+   manager before securely transmitting them to the user in question, and delete
+   them once the user has confirmed they can successfully log in.
+
+Even while following :ref:`passphrase best practices <passphrase_best_practices>`,
 your journalists may occasionally lock themselves out of their accounts. This
 can happen if, for example, they lose their two-factor device or if they
 forget the passphrase to their password manager. When this happens, you
-can reset their account by logging in as an administrator to the
-*Journalist Interface* and selecting *Admin* at the top right. Find their
-username and select **Edit**. Next, you can either rotate their passphrase or
-reset their two-factor authentication. You should make sure the user's
-passphrase is saved in a password manager *before* selecting the "Reset
-Password" button. To reset a user's two-factor authentication, choose the button
-that corresponds to the user's two-factor authentication method (hardware token,
-such as a Yubikey, or software-based application, such as FreeOTP).
+can reset their account as follows:
+
+* Log in as an administrator to the *Journalist Interface* and select *Admin* at
+  the top right to open the *Admin Interface*.
+
+* Find the user's account name and select **Edit**.
+
+Next, you can either rotate their passphrase or reset two-factor authentication
+for their account:
+
+* To change their passphrase to the randomly-generated passphrase shown, first
+  make sure the new passphrase is saved in a password manager, then select **Reset Password**.
+
+* To reset two-factor authentication, click the button that corresponds to the user's
+  chosen two-factor authentication method:
+
+  * Click **Reset Mobile App Credentials** for accounts using FreeOTP or a similar
+    authentication app.
+
+  * Click **Reset Security Key Credentials** for accounts using a Yubikey.
+
+* Follow the on-screen instructions to complete the process and verify their new
+  two-factor authentication credentials.
 
 |Reset Passphrase|
 
 Off-boarding Users
-~~~~~~~~~~~~~~~~~~
+^^^^^^^^^^^^^^^^^^
 
 See :doc:`our guide to off-boarding users from SecureDrop <offboarding>`.
 
-Server Command Line Use
-~~~~~~~~~~~~~~~~~~~~~~~
+Instance Configuration
+~~~~~~~~~~~~~~~~~~~~~~
+The Instance Configuration section of the *Admin Interface* allows you to:
+
+* update the organization name and logo displayed on the *Source* and *Journalist Interfaces*
+* set submission preferences for the *Source Interface*
+* send test OSSEC alerts.
+
+Updating the Organization Name
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Your organization name is used in page titles and logo ALT text on the
+*Source Interface* and *Journalist Interface*. By default, it's set to ``SecureDrop``.
+To change it, enter your desired name in the Organization Name field and click
+**SET ORGANIZATION NAME**.
```

Comment:
I would title-case this to ``**Set Organization Name*``. It's all-caps in the UI, but so are the other buttons which are all title-cased in this document.

Reply:
:white_check_mark: